docs/populate.md

```diff
@@ -10,18 +10,18 @@ import { compose } from 'redux'
 import { connect } from 'react-redux'
 import { firebaseConnect, populate } from 'react-redux-firebase'
 
-const populates = [
+const populatesOwnerWithUser = [
   { child: 'owner', root: 'users' } // replace owner with user object
 ]
 
 const enhance = compose(
   firebaseConnect([
     // passing populates parameter also creates all necessary child queries
-    { path: 'todos', populates }
+    { path: 'todos', populates: populatesOwnerWithUser }
```

Comment:
What does this renaming do here? Is it intended to more clearly indicate the name `populates`?